src/classes/CON_PrimaryAffls_TDTM.cls

```diff
@@ -105,21 +109,21 @@ public class CON_PrimaryAffls_TDTM extends TDTM_Runnable {
                         //         was his intention)
                         //  --> Make the old Affiliation non-primary. Although this could be AFFL_MultiRecordType_TDTM's responsibility, since the Affl type
                         //        is not yet defined (and the formula will run later), we'll do it here.
-                        else if(newContact.get(afflMapper.primaryAfflFieldNames[j]) != null && newContact.get(afflMapper.primaryAfflFieldNames[j])
-                        != oldContact.get(afflMapper.primaryAfflFieldNames[j])) {
+                        else if (newContact.get(afflMapper.primaryAfflFieldNames[j]) != null && newContact.get(afflMapper.primaryAfflFieldNames[j])
```

Comment:
@robertbwatson while you're in there, care to cuddle the else statements? they need love. or are they that way because of that comment? If so can we move the comments inside the else? I'm just having a really hard time reading that.

Reply:
@lifewithryan Yep, I'll make another quick pass at the formatting (cc @skosgi1 )

Reply:
Done